api_odoo_xmlrpc/wizard/api_odoo_synchro.py

```diff
@@ -0,0 +1,324 @@
+# See LICENSE file for full copyright and licensing details.
+
+import logging
+import time
+import threading
+from xmlrpc.client import ServerProxy
+
+from odoo import api, fields, models, _
+from odoo.exceptions import Warning
+
+_logger = logging.getLogger(__name__)
+
+
+class RPCProxyOne(object):
+
+    def __init__(self, server, ressource):
+        """Class to store one RPC proxy server."""
+        self.server = server
+        print ("Server URL =-=-", server.server_url)
```

Comment:
Please remove calls to print() throughout. Use _logger instead if it is something that is valuable to log, but if it's only for development/testing purposes please remove it.